sys/net/gnrc/network_layer/ipv6/gnrc_ipv6.c

```diff
@@ -772,6 +783,13 @@ static void _receive(gnrc_pktsnip_t *pkt)
     assert(netif);
     iface = ((gnrc_netif_hdr_t *)netif->data)->if_pid;
 
+#ifdef MODULE_NETSTATS_IPV6
+    assert(iface);
+    netstats_t *stats = gnrc_ipv6_netif_get_stats(iface);
+    stats->rx_count++;
+    stats->rx_bytes += (gnrc_pkt_len(pkt) - netif->size);
```

Comment:
But hasn't this a different semantic since the L2 addresses are not part of the struct?